nailgun/entities.py

```diff
@@ -5467,6 +5467,114 @@ def sync(self, synchronous=True, **kwargs):
         return _handle_response(response, self._server_config, synchronous)
 
 
+class ProductsBulkActions(Entity):
```

Comment:
class ProductsBulkAction(Entity):
all class names in nailgun has to be singular

Reply:
@ntkathole updated PR.

Reply:
@jameerpathan111 s/Actions/Action/

Reply:
updated PR.